pkg/load/configAgent.go

```diff
@@ -21,14 +21,15 @@ import (
 // memory and retrieve them when provided with a config.Info.
 type ConfigAgent interface {
 	GetConfig(config.Info) (api.ReleaseBuildConfiguration, error)
+	GetFilenameToConfig() FilenameToConfig
```

Comment:
nit: naming after types can be fragile, this is really `GetAll()` or `GetAllByFilename()`?